core/controllers/classroom.py

```diff
@@ -53,16 +54,13 @@ def get(self, classroom_name):
         if not constants.ENABLE_NEW_STRUCTURE_PLAYERS:
             raise self.PageNotFoundException
 
-        classroom_name_is_valid = False
-        for classroom_dict in config_domain.TOPIC_IDS_FOR_CLASSROOM_PAGES.value:
-            if classroom_dict['name'] == classroom_name:
-                classroom_name_is_valid = True
-                topic_ids = classroom_dict['topic_ids']
-                break
+        classroom_dict = classroom_services.get_classroom_dict_by_name(
```

Comment:
Can you make a Classroom domain object? In general, shouldn't pass dicts around.